spec/features/visitors/set_password_spec.rb

```diff
@@ -37,17 +41,20 @@
       confirm_last_user
     end
 
-    it 'updates strength feedback as password changes' do
-      expect(page).to have_content '...'
+    it 'does not have the password strength bar' do
+      fill_in t('forms.password'), with: 'howdy'
+      expect(page).to have_content(t('instructions.password.strength.intro'))
+    end
 
+    it 'updates strength feedback as password changes' do
       fill_in t('forms.password'), with: 'password'
```

Comment:
Related to previous comment, I think we could consolidate these so that they're run in a single test. Like we discussed in pairing, it's probably also duplicated with `user_profile_spec.rb` in case we'd want to go a step further and remove a duplicate.

```suggestion
    it 'updates strength feedback as password changes' do
      expect(page).not_to have_content(t('instructions.password.strength.intro'))
      
      fill_in t('forms.password'), with: 'password'
      expect(page).to have_content(t('instructions.password.strength.intro'))
```